pkgs/development/tools/misc/phpbench/default.nix

```diff
@@ -0,0 +1,34 @@
+{ stdenv, fetchurl, makeBinaryWrapper, lib, php80 }:
```

Comment:
Is there a reason to use `php80` over just `php`?

Reply:
`php81`  is throwing deprecations.

Reply:
for phpbench? it shouldn't be...

Reply:
I'll copy paste a screenshot tomorrow morning ;)